include/fplus/timed.hpp

```diff
@@ -124,16 +123,15 @@ namespace internal
 // API search type: make_timed_void_function : ((a -> Void)) -> (a -> Double)
 // fwd bind count: 0
 // Transforms a void function into a timed / benchmarked version of the same function.
-//
+// -
 // Example:
-//
-// void foo() {
-//     std::this_thread::sleep_for(std::chrono::milliseconds(1000));
-// }
+// -
+// void foo() { std::this_thread::sleep_for(std::chrono::milliseconds(1000)); }
 // ...
 // auto foo_bench = make_timed_void_function(foo);
 // auto r = foo_bench();
 // double run_time = foo_bench(); // in seconds
+// -
```

Comment:
You can remove this one line.The spacing between the documentation and the function declaration is added automatically by the frontend.